doc/reference/dune-workspace/pin.rst

```diff
@@ -0,0 +1,46 @@
+pin
+===
+
+.. warning::
+
+   Dune Package Management is not final yet and the configuration options
+   are subject to change.
+
+This stanza is used to define additional package sources.
+
+.. note::
+
+   Defining a pin does not enable it by default. It needs to be enabled in a
```

Comment:
Shouldn't it be more explicit that it needs to be in `(pins ...)` (IIUC)?

Reply:
I think it is fairly obvious, the stanza is not that large so I would defer if people find it unclear.